docs/standard/glossary.md

```diff
@@ -0,0 +1,228 @@
+
+* "The various .NET runtimes implement specific versions of .NET Standard."
+* "Libraries that are intended to run on multiple runtimes should target this framework." (referring to .NET Standard)
+* "The various .NET runtimes implement specific versions of .NET Standard. … Each .NET runtime version advertises the highest .NET Standard version it supports …"
+
+## stack
+
+A set of programming technologies that are used together to build and run applications. "The .NET stack" refers to the .NET Standard and all .NET implementations. "A .NET stack" may refer to one implementation of .NET. 
+
+## target framework
+
+The collection of APIs that a .NET app or library relies on. An app or library can target a version of .NET Standard (for example, .NET Standard 2.0), which is specification for a standardized set of APIs across all .NET implementations. An app or library can also target a specific .NET implementation, in which case it gets access to implementation-specific APIs. For example, an app that targets Xamarin.iOS gets access to Xamarin-provided iOS API wrappers.
+
+For some target frameworks (for example, the .NET Framework) the available APIs are defined by the assemblies that the framework installs on a system and may include app model APIs (for example, ASP.NET, Windows Forms). For package-based target frameworks (such as .NET Standard and .NET Core), the framework APIs are defined by the packages installed in the app or library. In that case the target framework implicitly specifies a metapackage that references all the packages that together make up the framework.
+
+See [Target Frameworks](frameworks.md).
+
+## TFM
+
+Target framework moniker. A standardized token format for specifying the target framework of a .NET app or library. Target frameworks are typically referenced by a Compact TFM (short name, such as net462). Long form TFMs (such as .NETFramework,Version=4.6.2) exist but are not generally used to specify a target framework.
+
+See [Target Frameworks](frameworks.md).
+
+## UWP
+
+Universal Windows Platform. An implementation of .NET that is used for building modern, touch-enabled Windows applications as well as headless devices for Internet of Things (IoT). It's designed to unify the different types of devices that you may want to target, including PCs, tablets, phablets, phones, and even the Xbox. UWP provides many services, such as a centralized app store, an execution environment (AppContainer), and a set of Windows APIs to use instead of Win32 (WinRT). Apps can be written in C++, C#, VB.NET, and JavaScript. When using C# and VB.NET the .NET APIs are provided by .NET Core.
+
+## See also
+
+[.NET Guide](index.md)  
+[.NET Framework Guide](../framework/index.md)  
+[.NET Core](../core/index.md)  
+[ASP.NET Overview](https://docs.microsoft.com/aspnet/index#pivot=aspnet)  
+[ASP.NET Core Overview](https://docs.microsoft.com/aspnet/index#pivot=core)  
```

Comment:
I'd suggest one more CR at the end of this line. It might not matter, but I get :skull: *death threats* :skull: from GH Desktop when I leave a file ending on a full line.